browser/misc_metrics/page_metrics_tab_helper.cc

```diff
@@ -26,13 +26,15 @@ PageMetricsTabHelper::PageMetricsTabHelper(content::WebContents* web_contents)
   page_metrics_ = ProfileMiscMetricsServiceFactory::GetServiceForContext(
                       web_contents->GetBrowserContext())
                       ->GetPageMetrics();
-  DCHECK(page_metrics_);
```

Comment:
should this maybe have CHECK_IS_TEST to preserve the nullptr check?

Reply:
Done in 9a2478ad33ecf80b8047fdae4494dd71776850d7